cluster/gce/gci/configure-helper.sh

```diff
@@ -1911,7 +1911,7 @@ function start-kube-controller-manager {
 function start-kube-scheduler {
   echo "Start kubernetes scheduler"
   create-kubeconfig "kube-scheduler" ${KUBE_SCHEDULER_TOKEN}
-  prepare-log-file /var/log/kube-scheduler.log
+  prepare-log-file /var/log/kube-scheduler.log ${KUBE_SCHEDULER_RUNASUSER:-0} ${KUBE_SCHEDULER_RUNASGROUP:-0}
```

Comment:
Try to go through pre-submit by setting the default to non-root.

Reply:
Done

Reply:
Assuming running as non-root passed GCE tests, please test with the following:
capabilities:
        drop:
        - all
      allowPrivilegeEscalation: false

Dropping all capabilities simply makes it explicit in the config that this container does not have any effective capabilities since it is running a non-root.

Reply:
Aren't these defined in the security context of the container and not the pod?

Reply:
On the securityContext, see https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container

Reply:
Ok all tests are passing, I'll test this as well.